src/Events/List/duck.js

```diff
@@ -9,9 +9,9 @@ const initialState = {
 };
 
 // Actions
-const REQUEST = 'conference/REQUEST';
-const RECEIVE = 'conference/RECEIVE';
-const FAIL = 'conference/FAIL';
+const REQUEST = 'events/REQUEST';
+const RECEIVE = 'events/RECEIVE';
+const FAIL = 'events/FAIL';
```

Comment:
These action names are very similar to the other action names:

```
const REQUEST = 'event/REQUEST';
const RECEIVE = 'event/RECEIVE';
const FAIL = 'event/FAIL';
```

This could be problematic due to 2 big reasons. Can you figure them out and tell me?